src/components/DataTable/DataTable.stories.js

```diff
@@ -3,14 +3,17 @@
  * @copyright IBM Security 2019 - 2020
  */
 
-import { boolean } from '@storybook/addon-knobs';
+import { boolean, select } from '@storybook/addon-knobs';
 import { storiesOf } from '@storybook/react';
 
 import { components } from '../../../.storybook';
 
+const sizes = [null, 'compact', 'short', 'tall'];
+
 const props = () => ({
-  useZebraStyles: boolean('Zebra row styles (useZebraStyles)', false),
+  size: select('Row height (size)', sizes, sizes[0]),
```

Comment:
Since sizes is just being used once, can the size values be included where they are referenced in line 14? In fact, I was wondering if this can be rewritten similar to:

https://github.com/carbon-design-system/carbon/blob/2692813cc53972ef4a460a51d87b7e45285ca742/packages/react/src/components/DataTable/DataTable-story.js#L17-L21